src/api.js

```diff
@@ -151,8 +163,9 @@ const Api = (options = {}) => {
         ...hdl
       ]
         .reduce((p, c) => p.then(c), Promise.resolve())
-        .then(async payload => generateResponse(null, payload, rb, {
-          defaultHeaders: await generateDefaultHeaders(event.headers)
+        .then(async ([payload, fields]) => generateResponse(null, payload, rb, {
```

Comment:
idk how I feel about formatting this (see changes below), unless you have a better way of passing the fields into the `generateResponse` method?